.github/workflows/build-and-push.yml

```diff
@@ -31,13 +31,31 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-    - uses: docker/build-push-action@v1
+    # https://github.com/docker/build-push-action/blob/master/docs/advanced/tags-labels.md
+    - name: Docker meta
+      id: meta
+      uses: crazy-max/ghaction-docker-meta@v2
+      with:
+        images: warsawlo/warsawlo-django
+        tags: |
+            type=ref,event=branch
+            type=sha
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
       with:
         username: ${{ secrets.DOCKER_USERNAME }}
         password: ${{ secrets.DOCKER_PASSWORD }}
-        repository: warsawlo/warsawlo-django
-        tag_with_ref: true
-        tag_with_sha: true
+
+    - name: Build and push
+      uses: docker/build-push-action@v2
+      with:
+        tags: ${{ steps.meta.outputs.tags }}
+        labels: ${{ steps.meta.outputs.labels }}
+        push: true
```

Comment:
noooooo!

Reply:
`false` pls, we don't need pull request images in the repo + this means pushing untrusted code to the repo which might end up running somewhere somehow

Reply:
Ok, this is the job that runs on master, sorry :ppp